### PR TITLE
feat: add command to close issues or prs

### LINKED ISSUE #424: Update Bot Message for GitHub
https://github.com/freeCodeCamp/discord-bot/blob/04b40191c97c007eab2bbc7b81a1be9448ba0f3e/src/commands/pullComment.ts#L17

Update the message to clarify that the repository name is required because it is scoped to the org.

### LINKED ISSUE #425: Add Bot Command to Close issues or PRs with a standard message
Add a bot command to close issues with the standard message.

### PR DESCRIPTION
**Pull request by naomi-lgbt**
Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #425 

Also closes #424 cause I was in here.

Adds a command which allows us to [close issues](https://github.com/freeCodeCamp/freeCodeCamp/issues/51601) or [pulls](https://github.com/freeCodeCamp/freeCodeCamp/pull/51488#issuecomment-1724175319) through Camperbot's discord account.

### CODE REVIEW

File: src/commands/close.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  data: new SlashCommandBuilder()
+    .setName("close")
+    .setDescription(
+      "Close an issue or pull request under the freeCodeCamp organisation."
+    )
+    .addStringOption((option) =>
+      option
+        .setName("repository")
+        .setDescription(
+          "The URL-friendly name of the repository to comment on. Must be a freeCodeCamp owned repository."
```
Comment (raisedadead) [inline src/commands/close.ts]:
```suggestion
          "The repository to comment on in freeCodeCamp/<repo> format."
```

File: src/commands/pullComment.ts [hunk 2]
```diff
@@ -9,12 +9,14 @@ export const pullComment: PrivilegedCommand = {
...
-    .setDescription("Adds a friendly comment to a pull request.")
+    .setDescription(
+      "Adds a friendly comment to a pull request. Scoped to the freeCodeCamp organisation."
+    )
     .addStringOption((option) =>
       option
         .setName("repository")
         .setDescription(
-          "The URL-friendly name of the repository to comment on."
+          "The URL-friendly name of the repository to comment on. Must be a freeCodeCamp owned repository."
```
Comment (raisedadead) [inline src/commands/pullComment.ts]:
```suggestion
          "The repository to comment on in freeCodeCamp/<repo> format."
```

**Review by raisedadead — CHANGES_REQUESTED** [review]
Nitpicking on the description, to make it clear

File: src/commands/close.ts (hunk 1 above)
Comment (naomi-lgbt) [reply src/commands/close.ts]: I worry that this reads like you have to pass `freeCodeCamp/discord-bot` as the repository value, when it should only be `discord-bot`.

Comment (raisedadead) [reply src/commands/close.ts]:
```suggestion
          "The name of the repository, under freeCodeCamp's GitHub org, to comment on"
```

Oh is it? Seeing the "URL-friendly name" got me confused.

File: src/commands/pullComment.ts (hunk 2 above)
Comment (naomi-lgbt) [reply src/commands/pullComment.ts]:
```suggestion
          "The name of the repository, under freeCodeCamp's GitHub org, to comment on"
```

**Review by raisedadead — DISMISSED** [review]

**Pull request merged**